hemamanne-plano.md

```diff
@@ -0,0 +1,7 @@
+# Plano, TX
+
+### Resturants
```

Comment:
Are there any Indian restaurants in Plano? 

If there are, are there ones worth visiting **OR** avoiding?